jenkins/debian/debos/scripts/strip.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/sh
+
+# Strip the image to a small minimal system without removing the debian
+# toolchain.
+
+set -e
+
+# Copy timezone file and remove tzdata package
+rm -rf /etc/localtime
+cp /usr/share/zoneinfo/Etc/UTC /etc/localtime
+
+
+UNNEEDED_PACKAGES=" libfdisk1"\
+" tzdata"\
+
+export DEBIAN_FRONTEND=noninteractive
+
+# Removing unused packages
+for PACKAGE in ${UNNEEDED_PACKAGES}
+do
+	echo ${PACKAGE}
+	if ! apt-get remove --purge --yes "${PACKAGE}"
+	then
+		echo "WARNING: ${PACKAGE} isn't installed"
+	fi
+done
+
+apt-get autoremove --yes || true
+
+# Removing unused files
+find . -name *~ -print0 | xargs -0 rm -f
```

Comment:
`"*~"` should be prefered to `*~`. If a filename ending with `~` was present, it would be matched by the shell, and then `find` would only find files of that name.